index.html

```diff
@@ -8,6 +8,181 @@
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     </head>
     <body>
-    
+      <!-- header -->
+      <header class="menu-nav">
+        <nav class="menu">
+            <p><a href="#home"><i></i>HOME</a></p>
```

Comment:
- [ ] Use a bolder font family
- [ ] text too light
- [ ] search is align at the other end of the page